__tests__/app.test.js

```diff
@@ -41,4 +42,40 @@ describe("app", () => {
         });
     });
   });
+  describe("/api/articles", () => {
+    it("200: GET - responds with an array of article objects in descending order by date and contains the correct properties", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          expect(articles).not.toHaveLength(0);
+          expect(articles).toBeSortedBy("created_at", {
+            descending: true,
+          });
+          articles.forEach((article) => {
+            expect(article).toHaveProperty("author", expect.any(String));
+            expect(article).toHaveProperty("title", expect.any(String));
+            expect(article).toHaveProperty("article_id", expect.any(Number));
+            expect(article).toHaveProperty("topic", expect.any(String));
+            expect(article).toHaveProperty("created_at", expect.any(String));
+            expect(article).toHaveProperty("votes", expect.any(Number));
+            expect(article).toHaveProperty(
+              "article_img_url",
+              expect.any(String)
+            );
+            expect(article).toHaveProperty("comment_count", expect.any(String));
+          });
+        });
+    });
+    it("404: GET - responds with 404 status code of Not Found when given an invalid endpoint", () => {
```

Comment:
You wont need to do this exact 404 test again as you have already done this in the previous ticket